### PR TITLE
fix: Scroll to given message

### PR DESCRIPTION
**Pull request by sravan-s**
Fixes:
* Scroll into view when startingpoint is set~
* Navigation to message from Thread and Search ~
* Scroll into message on clicking quote reply
* Some type warnings :)

fixes:
https://sendbird.atlassian.net/browse/UIKIT-3899
https://sendbird.atlassian.net/browse/UIKIT-3966
https://sendbird.atlassian.net/browse/UIKIT-3978

### CODE REVIEW

**Comment by sravan-s** [comment]
Kapture 2023-05-25 at 21 46 53

File: src/modules/Channel/components/Message/index.tsx [hunk 1]
```diff
@@ -205,6 +205,10 @@ const Message = ({
   if (renderedMessage) {
     return (
       <div
+        // donot delete this data attribute, used for scroll to given message
```
Comment (HoonBaek) [inline src/modules/Channel/components/Message/index.tsx:208]: typo?
`do not`

File: src/modules/Channel/context/utils.js [hunk 2]
```diff
@@ -8,6 +8,24 @@ import { OutgoingMessageStates } from '../../../utils/exports/getOutgoingMessage
 const UNDEFINED = 'undefined';
 const { SUCCEEDED, FAILED, PENDING } = getSendingMessageStatus();
 
+export const scrollToRenderedMessage = (scrollRef, initialTimeStamp) => {
```
Comment (HoonBaek) [inline src/modules/Channel/context/utils.js:11]: awesome, really appreciate

**Review by HoonBaek — APPROVED** [review]
LGTM

**Pull request merged**